src/index.html

```diff
@@ -11,6 +11,31 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="GET"
+      class="search-panel__big"
+      data-qa="big"
+    >
+      <input
+        data-qa="keypress"
+        class="search-bar search-bar--big"
```

Comment:
better add all style for big input (base input) to search-bar class and change style for small input